docker-images/python3.9.dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.9
+
+LABEL maintainer="Sebastian Ramirez <tiangolo@gmail.com>"
+
+RUN pip install --no-cache-dir uvicorn gunicorn
```

Comment:
In my experience we need to install uvicorn[standard] since uvloop etc is no longer included by default since 0.12+

Reply:
It seems you are right. I made the change and rebased to include https://github.com/tiangolo/uvicorn-gunicorn-docker/pull/54.